src/components/subheader/constants/subheader.general.js

```diff
@@ -0,0 +1,7 @@
+export const INITIAL_STATE = {
+    subheaderData : null,
```

Comment:
try to change the name of this, may be you can call it restaurantInfo or restaurantDetails